### PR TITLE
Marcela wip fe cards

### PR DESCRIPTION
**Pull request by marcelala**
styling of welcome banner, forms and postcard.
refactoring of nabber

### CODE REVIEW

File: frontend/src/styles/base/layout.css [hunk 1]
```diff
@@ -1,16 +1,69 @@
-* {
+*,
+*::after,
+*::after {
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:3]: ::after was doubled
Did you mean ::before?

File: frontend/src/styles/base/layout.css [hunk 2]
```diff
@@ -1,16 +1,69 @@
...
   margin: 0;
   padding: 0;
-  box-sizing: border-box;
+  box-sizing: inherit;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:6]: Why did we need this change?

File: frontend/src/styles/base/layout.css [hunk 3]
```diff
@@ -1,16 +1,69 @@
...
 }
 
-body {
+html,body {
   height: 100vh;
   width: 100vw;
-  font-family: "Thasadith", sans-serif;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: "Montserrat", sans-serif;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:15]: We need to decide which font-family we will have the main in the project

File: frontend/src/styles/base/layout.css [hunk 4]
```diff
@@ -1,16 +1,69 @@
...
   background-color: var(--color-beige);
 }
 
+body {
+  box-sizing: border-box;
+  position: relative;
+  padding: 0rem;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:22]: Padding 0 is already defined on the 5th line

File: frontend/src/styles/base/layout.css [hunk 5]
```diff
@@ -1,16 +1,69 @@
...
+}
+
+section {
+  padding: 1rem;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:26]: It's better not defined margins/paddings, text-align for section in common styles. It's very specific for different pages

File: frontend/src/styles/base/layout.css [hunk 6]
```diff
@@ -1,16 +1,69 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+p {
+  font-size: 1.6rem;
+}
+
+h2 {
+  font-size: 2rem;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/base/layout.css:54]: Not sure the font-sizes of these elements are the same on all pages

File: frontend/src/styles/modules/forms.css [hunk 7]
```diff
@@ -0,0 +1,65 @@
+input[type="text"],
```
Comment (OlgaPinchuk) [inline frontend/src/styles/modules/forms.css:1]: It's better to define the styles common for all the inputs in the app. Width, paddings, margins, font-sizes can vary

File: frontend/src/styles/modules/forms.css [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  box-sizing: border-box;
+  border-radius: 8px;
+  padding: 2rem;
+  max-width: 50rem;
+  border: none;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (OlgaPinchuk) [inline frontend/src/styles/modules/forms.css:60]: or a shorthand:
margin: 2rem auto;

File: frontend/src/styles/modules/navbar.css [hunk 9]
```diff
@@ -9,17 +9,17 @@
...
 }
 
-.header .logo {
+.nav-bar .logo {
     padding: 15px 0;
 }
 
-.header ul {
+.nav-bar ul {
     list-style: none;
```
Comment (OlgaPinchuk) [inline frontend/src/styles/modules/navbar.css:17]: this line can be deleted, list-style is defined in layout.css

File: frontend/src/styles/modules/postCard.css [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+  .postCard__content {
+    background: var(--color-beige);
+    margin-top: 4rem;
+    margin-bottom: 1rem;
+    margin-right: 2rem;
+    margin-left: 1rem;
+    padding: 1rem;
+    border-radius: 8px;
+    width: 95%;
+  }
```
Comment (OlgaPinchuk) [inline frontend/src/styles/modules/postCard.css:63]: or for better readability :
 margin: 4rem 2rem 1rem 1rem;

**Review by OlgaPinchuk — CHANGES_REQUESTED** [review]
Can you check the comments before merging

File: frontend/src/styles/base/layout.css (hunk 2 above)
Comment (marcelala) [reply frontend/src/styles/base/layout.css]: corrected

File: frontend/src/styles/base/layout.css (hunk 4 above)
Comment (marcelala) [reply frontend/src/styles/base/layout.css]: deleted

File: frontend/src/styles/modules/navbar.css (hunk 9 above)
Comment (marcelala) [reply frontend/src/styles/modules/navbar.css:17]: this code is Natalia's so I'm not gonna alter it

File: frontend/src/styles/modules/forms.css (hunk 7 above)
Comment (marcelala) [reply frontend/src/styles/modules/forms.css:1]: we can have this as standard and change for specific cases if/when necessary

**Pull request merged**